docs/content/getting-started/project-file-reference.mdx

```diff
@@ -0,0 +1,438 @@
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>__init__.py</td>
+      <td>
+        The <code>__init__.py</code> file includes a{" "}
+        <PyObject object="Definitions" /> object that contains all the definitions defined within your project. A definition can be an asset, a job, a schedule, a sensor, or a resource. This allows Dagster to load the definitions in an installed package.
+        <br />
+        Refer to the <a href="/concepts/code-locations#load-definitions-in-an-installed-package">Code locations documentation</a> to learn other ways to deploy and load your Dagster code.
+      </td>
+    </tr>
+    <tr>
+      <td>assets.py</td>
+      <td>
+        A Python module that contains {" "}
+        <a href="/concepts/assets/software-defined-assets">software-defined assets</a>.
+        <br />
+        <br />
+        <strong>Note:</strong> As your project grows, we recommend organizing assets in sub-packages or sub-modules.
+        For example, you can put all analytics-related assets in a <code>my_dagster_project/assets/analytics/</code>
+        folder and use <PyObject object="load_assets_from_package_module" /> in the top-level definitions
+        to load them, rather than needing to manually add assets to the top-level definitions every time
+        you define one. Similarly, you can also use <PyObject object="load_assets_from_modules" /> to
+        load assets from single Python files. Refer to the <a href="/guides/dagster/example_project">Fully Featured Project guide</a> for more info and best practices.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+        </td>
+    </tr>
+    <tr>
+      <td>my_dagster_project_tests/</td>
+      <td>
+        A Python package that contains tests for <code>my_dagster_project</code>
+        .
+      </td>
+    </tr>
+    <tr>
+      <td>README.md</td>
+      <td>A description and starter guide for the Dagster project.</td>
+    </tr>
+    <tr>
+      <td>pyproject.toml</td>
+      <td>
+        A file that specifies package core metadata in a static, tool-agnostic
+        way.
+        <br />
+        <br />
+        This file includes a <code>tool.dagster</code> section which references the
+        Python package with your Dagster definitions defined and discoverable at
+        the top level. This allows you to use the
+        <code>dagster dev</code> command to load your Dagster code without any
+        parameters. Refer to the{" "}
+        <a href="/concepts/code-locations">Code locations documentation</a> to
+        learn more.
+        <br />
+        <br /> <strong>Note:</strong> <code>pyproject.toml</code> was introduced
+        in{" "}
+        <a href="https://peps.python.org/pep-0518/https://peps.python.org/pep-0518/">
+          PEP-518
+        </a>{" "}
+        and meant to replace <code>setup.py</code>, but we may still include a{" "}
+        <code>setup.py</code> for compatibility with tools that do not use this
```

Comment:
Our current examples all use setup.py for dependencies, and deploying to cloud does not support specifying dependencies using pyproject.toml.

Maybe we could change the language a bit to replace this note and change say setup.py is used to specify dependencies?